mitodl/micromasters
# Updates to the program letter

#### Pre-Flight checklist


#### What are the relevant tickets?
Fix #4479

#### What's this PR do?
The following changes:
- [x] Allow for a wider logo graphic (but keep right alignment). This should accomodate the IDSS logo and address shown in the "desired letter" above
- [x] Move the closing "Cordially Yours" out of the template and into the letter content. This will allow the SDS team to use a different closing than SCM. 
  - [ ] when this commit is deployed, we will plan to make any necessary updates to the SCM-specific data
- [x] Review the spacing and image requirements for the signatures, so that three will fit in a vertical list, as displayed in the "desired letter" above. 
- [ ] would be nice: replace the current template footer (the name of the program) with another CMS image, similar to the header. 
- [x] would be nice: the current CMS form has a required field for institution, which doesn't appear in the letter. Can it be removed, or at least made not required? 
- [ ] any changes need to avoid regressions in the SCM letter, since that is already in production. 


#### How should this be manually tested?
Create a program letter and set up the CMS to be able to view it, enter the signatories and corresponding images. Make sure it looks as described in the issue.
